Workflow/Sources/AnyWorkflowConvertible.swift

```diff
@@ -38,7 +38,7 @@ extension AnyWorkflowConvertible {
     ///
     /// - Returns: The `Rendering` generated by the workflow.
     public func rendered<Parent>(in context: RenderContext<Parent>, key: String = "") -> Rendering where Output: WorkflowAction, Output.WorkflowType == Parent {
-        return asAnyWorkflow().render(context: context, key: key, outputMap: { AnyWorkflowAction($0) })
+        return asAnyWorkflow().render(context: context, key: key, outputMap: { $0 })
     }
 
     public func rendered<Parent, Action>(in context: RenderContext<Parent>, key: String = "", outputMap: @escaping (Output) -> Action) -> Rendering where Action: WorkflowAction, Action.WorkflowType == Parent {
```

Comment:
The line below this too, yeah? (github won’t let me comment on collapsed parts of code…)